<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerServer</class>
 <widget class="QMainWindow" name="PowerServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PowerServer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_online">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>在线用户</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_online">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_online">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>数量：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_online">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_offline">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>离线用户</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_offline">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>数量：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_offline">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTextBrowser" name="textBrowser_offline">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox_settings">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>服务器设定</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
         <property name="text">
          <string>120.78.235.195</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_IP">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>公网IP</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_port">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
         <property name="text">
          <string>10086</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_about">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
         <property name="text">
          <string>PowerChat Server - Insider Preview</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btn_quit">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>结束服务端程序</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_port">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>监听端口</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="btn_clear_attendance">
         <property name="text">
          <string>清除打卡记录</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_log">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>日志</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="textBrowser_log">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_admin">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>用户管理员</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="9" column="0">
        <spacer name="verticalSpacer">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>70</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="btn_addUser">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>添加用户</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="btn_save">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>保存用户数据</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_password">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_phone">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>手机</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_phone">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_email">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_deleteName">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="lineEdit_deleteName">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_email">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>邮箱</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_userName">
         <property name="font">
          <font>
           <family>Cascadia Code</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="2">
        <widget class="QPushButton" name="btn_deleteUser">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>删除该用户</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_userName">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>用户名</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QLabel" name="label_deleteUser">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>删除用户</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_addNew">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>添加新用户</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_userName</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_phone</tabstop>
  <tabstop>btn_addUser</tabstop>
  <tabstop>btn_save</tabstop>
  <tabstop>lineEdit_deleteName</tabstop>
  <tabstop>btn_deleteUser</tabstop>
  <tabstop>lineEdit_IP</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>btn_quit</tabstop>
  <tabstop>lineEdit_online</tabstop>
  <tabstop>textBrowser_online</tabstop>
  <tabstop>lineEdit_offline</tabstop>
  <tabstop>textBrowser_offline</tabstop>
  <tabstop>textBrowser_log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
